clarity/contracts/vaults-v2/arkadiko-vaults-operations-v1-1.clar

```diff
@@ -135,12 +137,12 @@
     (asserts! (not (var-get shutdown-activated)) (err ERR_SHUTDOWN))
     (asserts! (is-eq (get status vault) STATUS_ACTIVE) (err ERR_WRONG_STATUS))
     (asserts! (get valid coll-to-debt) (err ERR_INVALID_RATIO))
-    (asserts! (< (+ (- total-debt (get debt vault)) debt) (get max-debt collateral-info)) (err ERR_MAX_DEBT_REACHED))
-    (asserts! (>= debt (get vault-min-debt collateral-info)) (err ERR_MIN_DEBT))
+    (asserts! (< (+ (- total-debt (get debt vault)) new-debt) (get max-debt collateral-info)) (err ERR_MAX_DEBT_REACHED))
```

Comment:
Q/REQ: This is potentially an issue. We set caps and these can be hit, if we hit a cap, no one can update their vault anymore (except burning debt) cause the stability fees are constantly raising the debt amounts. Any thoughts on how to fix this?

Reply:
A: Right. We can just use `debt` instead of `new-debt` in this calculation to not take stability fees into account for the debt ceiling.

Reply:
Yeah indeed. Simply using `debt` instead of `new-debt` is the easy fix :)